lib/ansible/module_utils/powershell.ps1

```diff
@@ -205,6 +205,10 @@ Function Get-AnsibleParam($obj, $name, $default = $null, $resultobj = @{}, $fail
         if ($type -eq "path") {
             # Expand environment variables on path-type
             $value = Expand-Environment($value)
+            # Test if a valid path is provided
+            if (-not (Test-Path -IsValid $value)) {
```

Comment:
I did some messing around, and I haven't been able to find a situation where `LiteralPath` makes any difference in combination with `IsValid` (there are some real-world file situations where it does, but only when testing for presence, not validity). Wildcard/glob values aren't accepted either way (eg `C:\foo\bar*`), so any arg that accepts a wildcard can't be marked as type `path` regardless...